src/Endpoint/TokenEndpoint.php

```diff
@@ -59,4 +66,21 @@ protected function getRequestFormElements(): array
             'client' => $this->getConfigValue('client'),
         ];
     }
+
+    /**
+     * @inheritDoc
+     */
+    protected function getSerializer(): SerializerInterface
```

Comment:
The serializer for the Token needs to be different, as the endpoint uses snake_case